### PR TITLE
Allow gunicorn timeout setting to be set in pcconfig.py and environment variables

### LINKED ISSUE #1190: Allow gunicorn timeout setting to be set in pcconfig
Right now the only way to change gunicorn timeout setting if to manualy edit https://github.com/pynecone-io/pynecone/blob/fe89d53dcfcb467a69f2f585830a303607787afc/pynecone/constants.py#L108 It would be great if we could edit this setting from pcconfig.

### PR DESCRIPTION
**Pull request by ozeranskii**
### All Submissions:

- [x] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md) file?
- [x] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/pynecone-io/pynecone/pulls ) for the desired changed?

### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### New Feature Submission:

- [x] Does your submission pass the tests? 
- [x] Have you linted your code locally prior to submission?

### Changes To Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

### **After** these steps, you're ready to open a pull request.

    a. Give a descriptive title to your PR.

    b. Describe your changes.

    c. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

closes #1190

### CODE REVIEW

**Comment by ozeranskii** [comment]
The change requires an update of the documentation https://pynecone.io/docs/api-reference/config

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ To navigate our codebase with confidence, see [Pynecone Docs](https://pynecone.i
 
 - Have a question? Want to discuss a feature? [Start a discussion](https://github.com/pynecone-io/pynecone/discussions)
 
-    We welome and discussions and questions. We want to make sure that Pynecone is the best it can be, and we can't do that without your help.
+    We welcome and discussions and questions. We want to make sure that Pynecone is the best it can be, and we can't do that without your help.
```
Comment (Lendemor) [inline CONTRIBUTING.md:11]: If we're fixing typo, let's change the first `and` to  `any` too.
Good catch.

Comment (ozeranskii) [reply CONTRIBUTING.md]: Hi, fixed.

**Review by Lendemor — APPROVED** [review]
looks good

**Pull request merged**